docs/notebooks/formulary/coulomb.ipynb

```diff
@@ -0,0 +1,5759 @@
+       "Q 2700 2228 2247 2228 \n",
+       "L 1234 2228 \n",
+       "z\n",
+       "M 1606 4147 \n",
+       "L 1331 2741 \n",
+       "L 2272 2741 \n",
+       "Q 2775 2741 3058 2959 \n",
+       "Q 3341 3178 3341 3566 \n",
+       "Q 3341 3869 3150 4008 \n",
+       "Q 2959 4147 2541 4147 \n",
+       "L 1606 4147 \n",
+       "z\n",
+       "\" transform=\"scale(0.015625)\"/>\n",
+       "      </defs>\n",
+       "      <use xlink:href=\"#DejaVuSans-Oblique-42\" transform=\"translate(0 0.09375)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-6d\" transform=\"translate(68.603516 -16.3125)scale(0.7)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-69\" transform=\"translate(136.791992 -16.3125)scale(0.7)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-6e\" transform=\"translate(156.240234 -16.3125)scale(0.7)\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "    <g id=\"line2d_74\">\n",
+       "     <path d=\"M 412.955479 50.094688 \n",
+       "L 422.955479 50.094688 \n",
+       "L 432.955479 50.094688 \n",
+       "\" style=\"fill: none; stroke: #ff7f0e; stroke-width: 1.5; stroke-linecap: square\"/>\n",
+       "    </g>\n",
+       "    <g id=\"text_25\">\n",
+       "     <!-- $B_\\mathrm{min}$ -->\n",
+       "     <g transform=\"translate(440.955479 53.594688)scale(0.1 -0.1)\">\n",
+       "      <use xlink:href=\"#DejaVuSans-Oblique-42\" transform=\"translate(0 0.09375)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-6d\" transform=\"translate(68.603516 -16.3125)scale(0.7)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-69\" transform=\"translate(136.791992 -16.3125)scale(0.7)\"/>\n",
+       "      <use xlink:href=\"#DejaVuSans-6e\" transform=\"translate(156.240234 -16.3125)scale(0.7)\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"pa4e53a9051\">\n",
+       "   <rect x=\"43.78125\" y=\"22.318125\" width=\"299.225771\" height=\"215.6\"/>\n",
+       "  </clipPath>\n",
+       "  <clipPath id=\"p1befe3c911\">\n",
+       "   <rect x=\"403.955479\" y=\"22.318125\" width=\"299.225771\" height=\"215.6\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 720x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n_e = 1e25 * u.m ** -3\n",
+    "\n",
+    "dense_plasma = {\n",
+    "    \"T\": np.geomspace(1e5, 2e6) * u.K,\n",
+    "    \"n_e\": n_e,\n",
+    "    \"species\": [\"e-\", \"p+\"],\n",
+    "}\n",
+    "\n",
+    "plot_coulomb(**dense_plasma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8a7ab7e",
+   "metadata": {},
+   "source": [
+    "At low temperatures, $b_\\min$ is $ρ_⟂$. At higher temperatures, $b_\\min$ is $λ_{dB}$. What happens if we look at lower temperatures?"
```

Comment:
It would be nice to illustrate this by plotting rho and lamda on this same plot as dotted lines